game_env.py

```diff
@@ -897,6 +897,73 @@ def play(self):
 
         return winner
 
+    def create_board_reps(self, transition_list):
+        """
+        Returns a dictionary containing transition list of each transformation of the board
+        Total of 12 representation are possible 4 sides of the board X 3 views - normal, horizontal-flip
+        vertical - flip
+        # takes 3ms per transition list
```

Comment:
mention in this block what are the arguments of this function, just to confirm you are taking a single transition as input and not a list of transitions (the argument of this function suggests that a list of transitions is provided)